challenges/serializer-api.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+from .models import data
+from django.core.exceptions import ValidationError
+
+class RandnumbersSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = data
+        fields = ('id', 'numbers')
+
+    def validate_numbers(self, value):
+        queryset = data.objects.count()
+        if 500 != queryset:
+            raise serializers.ValidationError("You need to enter exactly 500 random numbers")
+        return value
+
+    def validate_numbers(self, value):
+        queryset = data.objects.count()
+        if 500 < queryset:
+            raise serializers.ValidationError("You need to enter exactly 500 random numbers")
+        return value
+
+    def validate_numbers(self, value):
+        queryset = data.objects.count()
+        if 500 > queryset:
+            raise serializers.ValidationError("You need to enter exactly 500 random numbers")
+        return value

```

Comment:
Only the last of these methods will be used by Django Rest Framework to validate the payload, because they all have the same name

Reply:
I wasn't sure how to validate the numbers within the JSON file so I just included the logic of how I was going to validate based on the instructions. I guess I could consolidate those methods into one and include a couple elifs. 

def validate_numbers(self, value):
        queryset = data.objects.count()
        
        if 500 != queryset:
            raise serializers.ValidationError("You need to enter exactly 500 random numbers")
            
        elif:
        if 500 < queryset:
            raise serializers.ValidationError("You need to enter exactly 500 random numbers")

        elif:
        if 500 > queryset:
            raise serializers.ValidationError("You need to enter exactly 500 random numbers")
            
        return value